src/StackNavigation.Uno.WinUI/Utils/Extensions/Microsoft.UI.Xaml.Media.Animation.Storyboard.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Threading.Tasks;
+
+namespace Microsoft.UI.Xaml.Media.Animation
+{
+    /// <summary>
+    /// Extension methods for classes in the Micrososft.UI.Xaml.Media.Animation namespace.
+    /// </summary>
+    internal static class StoryboardExtensions
```

Comment:
Couldn't this be shared? It feels like most of the could would be the same between UWP and WinUI?